File: orien.financebuddy.client/src/app/shared/Models/LoanDTO.ts
import { Banks } from "./Banks"
import { UserDetails } from "./UserDetails"

export interface Loans{
    Id: number
    Name: string
    Bank: Banks
    Amount: number
    Monthly_Emi: number
    AddedBy_UId: UserDetails
    CreatedDate: Date
    UpdatedDate: Date
}

File: Orien.FinanceBuddy.Business/Models/LoanDTO.cs
using Orien.FinanceBuddy.Data.Entity;

namespace Orien.FinanceBuddy.Business.Models
{
    public class LoanDTO
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public Bank Bank { get; set; }
        public decimal Amount { get; set; }
        public decimal Monthly_Emi { get; set; }
        public User AddedBy_UId { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
    }
}
